core/include/moveit/task_constructor/solvers/cartesian_path.h

```diff
@@ -57,7 +58,9 @@ class CartesianPath : public PlannerInterface
 	void setIKFrame(const std::string& link) { setIKFrame(Eigen::Isometry3d::Identity(), link); }
 
 	void setStepSize(double step_size) { setProperty("step_size", step_size); }
-	void setJumpThreshold(double jump_threshold) { setProperty("jump_threshold", jump_threshold); }
+	void setPrecision(const moveit::core::CartesianPrecision& precision) { setProperty("precision", precision); }
+	[[deprecated("Replace with setPrecision")]]  // clang-format off
```

Comment:
Let's not provide a no-op function for something that was rather essential before. :neutral_face: 
People tend to ignore deprecations and still assume things will work.
How about a [`static_assert` on use](https://stackoverflow.com/a/18814056/21260084)?

Reply:
Somewhat ugly, but ok to enforce deprecation.